src/Native/Runtime/RhConfig.cpp

```diff
@@ -51,7 +51,7 @@ UInt32 RhConfig::ReadConfigValue(_In_z_ const WCHAR *wszName, UInt32 uiDefaultVa
     {
         uiResult <<= 4;
 
-        WCHAR ch = wszBuffer[i];
+        TCHAR ch = wszBuffer[i];
         if ((ch >= L'0') && (ch <= L'9'))
```

Comment:
I have just removed the `L` prefixes for clarity. The default conversion will do the rest.